library/CMakeLists.txt

```diff
@@ -233,6 +233,10 @@ IF(UNIX)
   # Don't produce debug info for generated stubs
   SET_SOURCE_FILES_PROPERTIES(DataStatics.cpp DataStaticsCtor.cpp DataStaticsFields.cpp
                               PROPERTIES COMPILE_FLAGS "-g0 -O1")
+
+  # Don't emit thousands of warnings
+  SET_SOURCE_FILES_PROPERTIES(DataStaticsFields.cpp PROPERTIES COMPILE_FLAGS
+    "-Wno-missing-field-initializers -Wno-delete-non-virtual-dtor")
```

Comment:
It seems to pick one at random. I'll put both things for DataStaticsFields.cpp in the same SET_SOURCE_FILES_PROPERTIES.